portal-web/test/functional/com/liferay/portalweb/tests/enduser/webformsanddatalists/forms/fields/FormsDateField.testcase

```diff
@@ -31,6 +31,144 @@ definition {
 		}
 	}
 
+	@description = "Verify that is able to add a future validation using the response date as parameter"
+	@priority = "5"
+	test CanAddFutureValidationUsingResponseDate {
+		property forms.functional = "true";
+
+		var displayDateDay = DateUtil.getCurrentDate();
+		var displayDateMonth = DateUtil.getCurrentMonth();
+		var displayDateYear = DateUtil.getCurrentYear();
+		var futureYear = ${displayDateYear} + 1;
+		var pastYear = ${displayDateYear} - 1;
+		var displayFutureDate = "${displayDateMonth}/${displayDateDay}/${futureYear}";
+		var displayPastDate = "${displayDateMonth}/${displayDateDay}/${pastYear}";
+		var futureHalfYear = StringUtil.extractLast("${futureYear}","0");
+
+		FormsAdmin.addForm();
+
+		Form.editName(formName = "Form 1");
+
+		Form.gotoAddField(
+			fieldPositionNumber = "0",
+			fieldType = "Date");
+
+		Form.editFieldLabelMetal(fieldValue = "Date Field");
+
+		Form.gotoAdvancedTab();
+
+		FormFields.enableSwitch(fieldName = "Validation");
+
+		FormFields.validationRangeCustomDate(acceptedDateOption = "Future Dates");
+
+		FormFields.validationCustomDateStartsFrom(
+			startsFromOperation = "Plus",
+			startsFromOption = "Custom Date",
```

Comment:
@rafaellajordao, this test intends to check **Response Date** however the test is checking **Custom Date**, can you please adjust it?

Also, please check my previous comment regarding the macro's names and try to apply here.